drivers/backup/backup.go

```diff
@@ -309,3 +310,11 @@ func Get(name string) (Driver, error) {
 		Type: "BackupDriver",
 	}
 }
+
+func init() {
+	str, err := GetPxCentralAdminPwd()
+	if err != nil {
+		logrus.Errorf("Error fetching password from secret: %v", err)
+	}
```

Comment:
Let's also exit from here so that test aborts.
The rest looks good.

Reply:
Adding `panic()` for this.